tests/testthat/error/test-type-data-frame.txt

```diff
@@ -6,23 +6,23 @@ combining data frames with foreign classes uses fallback
 > bar <- structure(mtcars[4:6], class = c("bar", "data.frame"))
 > baz <- structure(mtcars[7:9], class = c("baz", "data.frame"))
 > vec_ptype_common(foo, bar, baz)
-Warning: Can't combine <foo> and <bar>.
+Warning: Can't combine <foo[,3]> and <bar[,3]>.
```

Comment:
What does `[,3]` mean?

Reply:
Oh it's the number of columns? I find this super confusing so I think you should just drop it, and return to the previous `vec_ptype_abbr()`.

Reply:
Should we drop it from errors too? Right now I've just made it consistent with errors.

Reply:
Though I guess errors returned the full column spec until recently.

Reply:
Yeah, exactly.